integration-tests/src/test/resources/wdt-models/model.delete.sysresourcesbyconfigmap.yaml

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2020, 2022, Oracle and/or its affiliates.
```

Comment:
Modify the Copyright statement to include year 2023

Reply:
Done